src/transformers/models/llama/modeling_llama.py

```diff
@@ -181,6 +184,8 @@ def apply_rotary_pos_emb(q, k, cos, sin, position_ids, unsqueeze_dim=1):
     Returns:
         `tuple(torch.Tensor)` comprising of the query and key tensors rotated using the Rotary Position Embedding.
     """
+    cos = cos.unsqueeze(unsqueeze_dim)
+    sin = sin.unsqueeze(unsqueeze_dim)
```

Comment:
let's unsqueeze in the rotary embedding no? or that changes the shape we previously had?

Reply:
Same shapes, but unsqueezing here is preferable by some users (see https://github.com/huggingface/transformers/pull/27117)